specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2025-02-01/examples/Capabilities.json

```diff
@@ -15,7 +13,9 @@
           }
         ]
       }
-    }
+    },
+    "location": "westus",
```

Comment:
>location [](http://example.com/codeflow?start=5&length=8)

Probably unimportant but I'm not clear why the example needs to involve changing the ordering so much. Is it alphabetizing things when it doesn't really need to?

Reply:
Please ask in TypeSpec Discussion channel for this question.